src/pre_process/m_global_parameters.fpp

```diff
@@ -216,6 +216,8 @@ module m_global_parameters
     type(pres_field) :: pb
     type(pres_field) :: mv
 
+    integer :: max_iter_pc_ts !< maximum number of iterations for phase change at each time-step    
```

Comment:
Given the `maxval` above, the (user-specified) input-variable would actually be a lower-bound for this value. Could be useful to separate these two for clarity.

Reply:
This is not user-specified, as commented above